src/agent/index.ts

```diff
@@ -118,4 +138,5 @@ export const sendSiopAuthorizationResponse = agent.sendSiopAuthorizationResponse
 export const keyManagerSign = agent.keyManagerSign
 export const dataStoreGetVerifiableCredential = agent.dataStoreGetVerifiableCredential
 export const dataStoreDeleteVerifiableCredential = agent.dataStoreDeleteVerifiableCredential
+export const createVerifiableCredentialLDLocal = agent.createVerifiableCredentialLDLocal
```

Comment:
We really should use the Veramo methods and not directly call the LD local method. In the future we also want to be able to issue JWT credentials, which ICredentialIssuer/Veramo already supports. The only difference is that you will have to add a param that denotes the VC type (lds for JsonLD and jwt for JWT)